lib/chef_apply/startup.rb

```diff
@@ -71,9 +75,9 @@ def run
     rescue ConfigPathInvalid => e
       UI::Terminal.output(T.error.bad_config_file(e.path))
     rescue ConfigPathNotProvided
-      UI::Terminal.output(T.error.missing_config_path)
+      UI::Terminal.output(T.error.missing_config_path(D::RUNEXEC, D::DK))
```

Comment:
This is definitely appropriate looking at the error message, but internally we have a [ticket](https://github.com/chef/chef-workstation/issues/689) to change this from `.chefdk` to `.chef-workstation`. We'll come back and clean this up 😄